gensrc/thrift/PlanNodes.thrift

```diff
@@ -1063,6 +1072,12 @@ struct THdfsScanNode {
     17: optional Types.TTupleId mor_tuple_id;
 }
 
+struct TKuduScanNode {
+    1: optional Types.TTupleId tuple_id
+    // table name it scans
+    2: optional string table_name;
+}
```

Comment:
why not just use hdfsscannode, how should we transfer predicates to be?

Reply:
Actually, I utilized THdfsScanNode, KuduScanNode is redundant. And predicate transfer is handled via THdfsScanRange#kudu_scan_token.